reactive/webserver/cors/src/main/java/module-info.java

```diff
@@ -14,10 +14,19 @@
  * limitations under the License.
  */
 
+import io.helidon.common.features.api.Feature;
+import io.helidon.common.features.api.HelidonFlavor;
+
 /**
  * The Helidon SE CORS module
  */
+@Feature(value = "CORS",
+        description = "CORS support for WebServer",
+        in = HelidonFlavor.SE,
+        path = {"CORS support for WebServer"})
```

Comment:
Path should be `WebServer, CORS` as it was in feature catalog

Reply:
Sorry. Typo.. Fixed.